.github/workflows/manylinux/entrypoint.sh

```diff
@@ -26,8 +26,8 @@ mkdir build_source && cd build_source
 
 cmake ../cmake -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_POSITION_INDEPENDENT_CODE=ON -Dprotobuf_BUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=Release
 make -j$(nproc)
-sudo make install
-sudo ldconfig
+make install
+ldconfig
```

Comment:
Probably you should completely remove the "ldconfig" line. 

You have set BUILD_SHARED_LIBS=OFF. If there is no shared libraries installed, then you don't need to run ldconfig.